app/models/item.rb

```diff
@@ -7,11 +7,12 @@ class Item <ApplicationRecord
   validates_presence_of :name,
                         :description,
                         :price,
-                        :image,
                         :inventory
-  validates_inclusion_of :active?, :in => [true, false]
-  validates_numericality_of :price, greater_than: 0
+  validates_presence_of :image, :allow_blank => true
 
+  validates_inclusion_of :active?, :in => [true, false]
+  validates_numericality_of :price, greater_than: 0, message: "Price must be greater than 0"
+  validates_numericality_of :inventory, greater_than_or_equal_to: 0, message: "Inventory must be greater than or equal to 0"
```

Comment:
Very explicit constraints, i like this a lot

Reply:
Yes, I also think these are very well written.